topostats/grains.py

```diff
@@ -610,7 +655,15 @@ def improve_grain_segmentation_unet(
         # You may also get an error referencing a "group_1" parameter, this is discussed in this issue:
         # https://github.com/keras-team/keras/issues/19441 which also has an experimental fix that we can try but
         # I haven't tested it yet.
-        unet_model = keras.models.load_model(unet_config["model_path"], custom_objects={"mean_iou": mean_iou})
+
+        # Important for when loading the model inievitably fails on users' machines.
```

Comment:
How does this stack up against the recent changes which reduce logging output?

Could we perhaps have these reported as `LOGGER.error()` within a `try:... except: ...` when the failure occurs?

Reply:
Good idea, will do!